### DISCUSSION TITLE
Coding on an iPad using code-server and Railway

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#3620] [Show and tell]
I published a new video on YouTube which walks you through deploying code-server to Railway and using as a PWA on your iPad 🚀

https://www.youtube.com/watch?v=IWfkkaY5fXs

**Comment by hao-hao-hao** [comment]
Thanks for the guide. May I ask how do you preview the webpage by using railway? for example if I want to build an react web app, how do I check the actual web page I built?

**Reply by lg-nineoneninethreenine** [reply]
Well, you can't preview it on Railway, you have to visit the site where your React App is hosted. If you are asking how to see from within code-server, you have to try this:
https://{{your-url-of-railway}}/proxy/{port}/
Remember to put a slash at the end or it won't work.

**Reply by hao-hao-hao** [reply]
That works, thanks a lot for the information.

**Comment by teja-self-taught-cs** [comment]
It says u can't run live server from ipad which is a web environment... Any solution?
